### PR TITLE
marking provision tests

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================
Marking Provisioning tests

{{pytest:  --use-provider ec2east cfme/tests/cloud/test_provisioning.py::test_provision_from_template}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 48538e579d606a8a826cd6546f9920c8494eb6f9:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/cloud/test_provisioning.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def vm_name(request):
     return vm_name
 
 
+@pytest.mark.requirement('provision')
```
Comment (psav) [inline cfme/tests/cloud/test_provisioning.py]: Please can this be replaced wit hthe newer marking method mentioned in the latest email?

**Comment by dajoRH** [comment]
Lint report for commit e51e5c18e7767e7567c66c9ec3b5bc0e0ed680d7:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by lkhomenk** [comment]
@psav I've update. but this PR depends on REST tests. there're n open PR from Martin for REST API test. but question is the same - should we have double marks?

**Pull request merged**